cmd/osbuild-tests/main.go

```diff
@@ -71,13 +71,22 @@ func main() {
 }
 
 func testCompose(outputType string) {
+	tmpdir := NewTemporaryWorkDir("osbuild-tests-")
+	log.Println(tmpdir.Path)
+	defer tmpdir.Close()
+
 	bp := blueprint.Blueprint{
 		Name:        "empty",
 		Description: "Test empty blueprint in toml format",
 	}
 	pushBlueprint(&bp)
 	defer deleteBlueprint(&bp)
 
+	runComposerCLI(false, "blueprints", "save", "empty")
```

Comment:
Hm, this hunk is unrelated to this patch, no?

Reply:
Indeed. I've split it into a separate commit.